---
layout: post
title: Taking the 6 Week Samples for the DiNV Experimental Evolution and DNA Extractions 
---

## Taking Samples and Fluid Changing Flasks at the 6 Week Mark for Experimental DiNV Evolution in Dinn Cells (~42 days) and Extracting DNA from 1 50ul Sample Per Replicate Flask, and Checking Samples for Virus Presence

**Sampling 20231107**

Cells were inoculated on the 26th of Sept [see post here](https://meschedl.github.io/Unckless-Lab-Notebook-Maggie/2023/09/26/experimental-evolution-inoculation.html). There are 3 replicate flasks that have been left to grow and get infected in the 23C incubator for 42 days. I am only removing supernatant for samples, the cells are not getting passed throughout this experiment, the only thing that happens is a tri-weekly fluid change (once every three weeks). In the [three week sampling](https://meschedl.github.io/Unckless-Lab-Notebook-Maggie/2023/10/17/experimental-evolution-3-week-samples.html) I only took 2 50ul samples from each replicate flask. Because supernatant is the only thing I am sampling, there is a really small amount of DNA in there. I used a lot of the DNA from one of those samples just getting the PCR to run to confirm virus presence. Ideally these samples will need to be used for multiple PCRs (9 if we are going to use the [primers around SNPs](https://meschedl.github.io/Unckless-Lab-Notebook-Maggie/2023/11/02/Testing-Experimental-Evolution-Primers.html)) as well as WGS sequencing. I am not sure there is enough DNA in 2 samples for that to happen. So I decided to greatly increase the number of samples I take, 7 50ul supernatant and 1 1mL supernatant sample per replicate flask. 

All steps took place in the cell culture hood 

- Made fresh 10% FBS, 4% mushroom Schneider's medium with antibiotics in the hood to room temperature 
- Prepared 1.5mL tubes to take samples 
- Removed 7 samples of 50ul and 1 sample of 1000ul from each flask before changing fluid and placed into the 1.5mL tubes:

| tube number | day sampled | date sampled | sample volume | replicate tube | flask from |
|-------------|-------------|--------------|---------------|----------------|------------|
| 17          | day 42      | 20231107     | 50ul          | 1              | A          |
| 18          | day 42      | 20231107     | 50ul          | 2              | A          |
| 19          | day 42      | 20231107     | 50ul          | 3              | A          |
| 20          | day 42      | 20231107     | 50ul          | 4              | A          |
| 21          | day 42      | 20231107     | 50ul          | 5              | A          |
| 22          | day 42      | 20231107     | 50ul          | 6              | A          |
| 23          | day 42      | 20231107     | 50ul          | 7              | A          |
| 24          | day 42      | 20231107     | 1mL           | NA             | A          |
| 25          | day 42      | 20231107     | 50ul          | 1              | B          |
| 26          | day 42      | 20231107     | 50ul          | 2              | B          |
| 27          | day 42      | 20231107     | 50ul          | 3              | B          |
| 28          | day 42      | 20231107     | 50ul          | 4              | B          |
| 29          | day 42      | 20231107     | 50ul          | 5              | B          |
| 30          | day 42      | 20231107     | 50ul          | 6              | B          |
| 31          | day 42      | 20231107     | 50ul          | 7              | B          |
| 32          | day 42      | 20231107     | 1mL           | NA             | B          |
| 33          | day 42      | 20231107     | 50ul          | 1              | C          |
| 34          | day 42      | 20231107     | 50ul          | 2              | C          |
| 35          | day 42      | 20231107     | 50ul          | 3              | C          |
| 36          | day 42      | 20231107     | 50ul          | 4              | C          |
| 37          | day 42      | 20231107     | 50ul          | 5              | C          |
| 38          | day 42      | 20231107     | 50ul          | 6              | C          |
| 39          | day 42      | 20231107     | 50ul          | 7              | C          |
| 40          | day 42      | 20231107     | 1mL           | NA             | C          |

- Removed 1.5mL of remaining medium in each flask and discarded it into a 15mL conical. The conical was placed in the autoclave trash when full (should contain virus)
- Added 3mL of 10% FBS, 4% mushroom Schneider's medium with antibiotics to each flask gently
- Gently rocked flasks to distribute the medium 
- Placed flasks back in the 23C incubator for another 3 weeks 
- Sample tubes were frozen at -20  

Sample information can be found [here](https://docs.google.com/spreadsheets/d/1balXyQJFsFGfXK0ooPFMvkFJZwOlg1-xvkuuOEXaU14/edit#gid=0)

**DNA Extractions 20231108**

For these DNA extractions I added in a second tube of fly homogenate, and to balance everything (because it was an uneven number) I added a second replicate A extraction. For the fly homogenate, I thawed an aliquot (I think that had already been thawed and used once), removed 50ul to a sample tube, and froze what was left. I am not too concerned about the freeze thaws because the DNA should be relatively intact. 

Samples: 

|sample #|day|flask|
|17|day 42|A|
|23|day 42|A|
|25|day 42|B|
|33|day 42|C|
|41| NA|fly homogenate|


All samples were thawed on ice, [this general protocol](https://github.com/meschedl/Unckless_Lab_Resources/blob/main/protocols/cell-DNA-extraction-protocol.md) was used, and filter tips were used at all times. Since all samples were presumed positive for DiNV, there was no specific order to them. 2 Extraction controls were done at the same time as well. 

**PCRs 20231116**

Each sample was diluted 1:1 in water for PCR to conserve the sample. I did not run sample 23 because it was excess. The positive control was not diluted. The PCR programs were set to 30 cycles, but these might need to be changed in the future because of the low DNA level. Information on the primers and their programs can be found [here](https://docs.google.com/spreadsheets/d/1IaLLjsa4SXJr90wUi8xyE1dYvWmHsbThSz3d8N9KaK0/edit#gid=0), but the TPI program had an extension time of 30 seconds. 

The PCR process followed this [general protocol](https://github.com/meschedl/Unckless_Lab_Resources/blob/main/protocols/PCR_protocol_general.md). Afterwards a 1% gel was run for 25 minutes at 100V. There are a few other samples on this gel that I was running at the same time. The samples from this project ad ex ctr 3, ex ctr 4, 17, 25, 33, and 41. 

![](https://raw.githubusercontent.com/meschedl/Unckless-Lab-Notebook-Maggie/master/images/20231116-gel.jpeg)
